{"uuid": "e5e57067-e5df-496e-a69d-e00ea84f35e6", "children": ["a1dfaca5-00a9-4e62-a6ad-f53c7c36f3eb"], "befores": [{"name": "driver", "status": "passed", "start": 1693403581139, "stop": 1693403585356}], "afters": [{"name": "driver::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidArgumentException: Message: Must either pass base64 data or a local file path\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/allure_commons/_allure.py\", line 209, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/_pytest/fixtures.py\", line 916, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/yaroslav.petryk/Desktop/projects/testrail_android/tests/conftest.py\", line 21, in driver\n    appium_driver.push_file('./tests/config.json')\n  File \"/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/appium/webdriver/extensions/remote_fs.py\", line 80, in push_file\n    raise InvalidArgumentException('Must either pass base64 data or a local file path')\n"}, "start": 1693403607896, "stop": 1693403607909}], "start": 1693403581139, "stop": 1693403607909}